### PR TITLE
[CBRD-23328] shutdown load sessions

### PR DESCRIPTION
**Pull request by ribram**
http://jira.cubrid.org/browse/CBRD-23328

use load worker manager to track registered sessions and interrupt all of them on shutdown.

### CODE REVIEW

File: src/loaddb/load_worker_manager.cpp [hunk 1]
```diff
@@ -166,6 +174,27 @@ namespace cubload
...
+  {
+    std::unique_lock<std::mutex> ulock (g_wp_mutex);
+    if (g_active_sessions.empty ())
+      {
+	return;
+      }
+
+    for (auto &it : g_active_sessions)
+      {
+	it->interrupt ();
```
Comment (valerinistor) [inline src/loaddb/load_worker_manager.cpp]: I think session needs to be deleted for this case

Comment (ribram) [reply src/loaddb/load_worker_manager.cpp]: won't load session be deleted by either sloaddb_destroy or session_state_uninit?

Comment (valerinistor) [reply src/loaddb/load_worker_manager.cpp]: in case we are sure that one of these functions is called on shutdown then it is safe

Comment (ribram) [reply src/loaddb/load_worker_manager.cpp]: I think it is safer than deleting here; I am not sure load client will expect a deleted session.

**Pull request merged**